packages/mongodb-chatbot-verified-answers/.env.example

```diff
@@ -1,6 +1,6 @@
 MONGODB_DATABASE_NAME=<docs-chatbot-ENV>
 MONGODB_CONNECTION_URI=<connection_uri>
-OPENAI_EMBEDDING_MODEL=text-embedding-ada-002
+OPENAI_EMBEDDING_MODEL_NAME=text-embedding-ada-002
```

Comment:
can this just be removed?

Reply:
rather, we should use "OPENAI_EMBEDDING_DEPLOYMENT" for consistency w/ other packages

Reply:
https://jira.mongodb.org/browse/EAI-502